python/ray/rllib/common.py

```diff
@@ -1,15 +1,44 @@
 from collections import namedtuple
 from datetime import datetime
-import json
 import logging
 import os
+import simplejson
```

Comment:
Yeah, it is now using json with a custom JSONEncoder, I think that's the cleanest solution! Thanks for the hint!